staging/src/k8s.io/api/admissionregistration/v1/types.go

```diff
@@ -563,3 +579,25 @@ type ServiceReference struct {
 	// +optional
 	Port *int32 `json:"port,omitempty" protobuf:"varint,4,opt,name=port"`
 }
+
+// MatchCondition represents a condition which must by fulfilled for a request to be sent to a webhook.
+type MatchCondition struct {
+	// Name is an identifier for this match condition, used for strategic merging of MatchConditions,
+	// as well as providing an identifier for logging purposes. A good name should be descriptive of
+	// the associated expression.
+	// Name must be a valid RFC 1123 DNS subdomain, and unique in a set of MatchConditions.
+	// Required.
+	Name string `json:"name" protobuf:"bytes,1,opt,name=name"`
+	// NOTE: Placeholder documentation, to be replaced by https://github.com/kubernetes/website/issues/39089.
+	//
+	// Expression represents the expression which will be evaluated by CEL.
+	// ref: https://github.com/google/cel-spec
+	// CEL expressions have access to the contents of the AdmissionRequest, organized into CEL variables:
+	//
+	// 'object' - The object from the incoming request. The value is null for DELETE requests.
+	// 'oldObject' - The existing object. The value is null for CREATE requests.
+	// 'request' - Attributes of the admission request([ref](/pkg/apis/admission/types.go#AdmissionRequest)).
+	//
+	// Required.
+	Expression string `json:"expression" protobuf:"bytes,2,opt,name=expression"`
```

Comment:
Worth to refer to existing doc(or even direct copy maybe): https://github.com/kubernetes/kubernetes/blob/44b2fb4cfaf99d7cbe3fc385bf82b182e9ab8bf5/staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go#L139-L173